MartinP460/react-denmark-map
# Add Prettier configuration file

### Proposed changes

Added a `.prettierrc` configuration file for Prettier. This ensures that people running Prettier locally use this configuration instead of their local. Also formatted a few files to be in line with the new configuration.

### How to test

1. Run "npm run format" from root and make sure that it is able to run Prettier.
